spec/latest/json-ld-api/index.html

```diff
@@ -876,7 +877,7 @@
         <li>If <em>value</em> contains the key <code>@type</code>:
           <ol class="algorithm">
             <li>Initialize <em>type</em> to the value associated with the
-              <code>@type</code> key, which must be a <a>string</a>. Otherwise, an
+              <code>@type</code> key, which MUST be a <a>string</a>. Otherwise, an
```

Comment:
Debatable, but that seems to be were we came down in the 1.0 document, so we'll use "must".